src/components/fieldset/template.test.js

```diff
@@ -96,16 +92,41 @@ describe('fieldset', () => {
     expect($headingInsideLegend.text().trim()).toBe('What is your address?')
   })
 
-  it('renders attributes', () => {
+  it('can have additional classes on the legend', () => {
+    const $ = render('fieldset', {
+      legend: {
+        text: 'What is your address?',
+        classes: 'my-custom-class'
+      }
+    })
+
+    const $legend = $('.govuk-fieldset__legend')
+    expect($legend.hasClass('my-custom-class')).toBeTruthy()
+  })
+
+  it('can have additional classes on the fieldset', () => {
+    const $ = render('fieldset', {
+      classes: 'app-fieldset--custom-modifier'
+    })
+
+    const $component = $('.govuk-fieldset')
+    expect($component.hasClass('app-fieldset--custom-modifier')).toBeTruthy()
+  })
+
+  it('can have additional attributes', () => {
     const $ = render('fieldset', {
       attributes: {
-        'data-attribute': 'value',
-        'data-another-attribute': 'another-value'
+        'data-attribute': 'value'
       }
     })
 
     const $component = $('.govuk-fieldset')
     expect($component.attr('data-attribute')).toEqual('value')
-    expect($component.attr('data-another-attribute')).toEqual('another-value')
+  })
+
+  it('renders nested components using `call`', () => {
```

Comment:
Minor, but can we move this above the tests for additional classes and additional attributes? Trying to get the 'core' component tests (the things you need to be able to do use the component) at the top, followed by the tests that allow you to 'extend' a component.